database/migrations/20200208123841_SKILLED_IMPACT_REQUESTS.js

```diff
@@ -0,0 +1,68 @@
+const skills = [
+    'ACCOUNTING',
+    'ARCHITECTURE',
+    'AUTO',
+    'APP_DEVELOPMENT',
+    'ART',
+    'AVIATION',
+    'BOATING',
+    'BUSINESS_DEVELOPMENT',
+    'COMMUNICATION',
+    'COMMUNITY_LIAISON',
+    'CONSTRUCTION',
+    'CRAFT',
+    'CULINARY',
+    'DATA_ANALYSIS',
+    'DATABASE_MANAGEMENT',
+    'DIVING',
+    'DRONE',
+    'ELECTRICITY',
+    'ENGINEERING',
+    'ENTREPRENEURSHIP',
+    'FINANCE',
+    'FUNDRAISING',
+    'GAMING',
+    'GRAPHIC_DESIGN',
+    'HOSPITALITY',
+    'HUMAN_RESOURCES',
+    'INFORMATION_TECHNOLOGY',
+    'LANDSCAPE',
+    'LEGAL',
+    'MANAGEMENT',
+    'MARKETING',
+    'MEDICAL',
+    'MUSICAL',
+    'PHOTOGRAPHY',
+    'PLUMBING',
+    'PUBLIC_RELATIONS',
+    'RENEWABLE_ENERGY',
+    'RESEARCH',
+    'SOCIAL_MEDIA',
+    'STRATEGY_CONSULTING',
+    'TAXI',
+    'TRANSLATION',
+    'VETERINARY_SERVICES',
+    'VIDEOGRAPHY',
+    'WEB_DESIGN',
+    'WEB_DEVELOPMENT',
+    'WRITING'
+];
+exports.up = function(knex) {
+    return knex.schema.createTable('skilled_impact_requests', table => {
+        table.increments('id').notNullable().unique();
+        table.integer("campaign_id").notNullable()
+            .references("campaigns.camp_id")
+            .onDelete("CASCADE")
+            .onUpdate("CASCADE");
+        table.enu("skills",skills,{ useNative: true, enumName: 'enum_skills' }).notNullable();
```

Comment:
Let's change this to "skill" (singular) since each skilled impact request corresponds to a single skill.